gulp-options.js

```diff
@@ -21,6 +21,7 @@ options.rootPath = {
 
 options.theme = {
   name: 'kaizen',
+  root : options.rootPath.project,
```

Comment:
maybe better to add specific options.theme.svg ?

Reply:
fixed